Source/WebKit/Scripts/webkit/tests/SerializedTypeInfo.cpp

```diff
@@ -461,6 +461,12 @@ Vector<SerializedTypeInfo> allSerializedTypes()
                 "pendingReads()"_s
             },
         } },
+        { "Namespace::OuterClass"_s, {
```

Comment:
I feel like this should have ENABLE(OUTER_CONDITION) around it.

Reply:
For some reason, `[Nested]` class declarations don't reflect this? Changed to a bare class declaration for now, I'm not sure what's special about `[Nested]`.